### DISCUSSION TITLE
subject and examiner profile

### DISCUSSION THREAD

**Discussion by siamakhaz** [ethierlab/Motopya#15] [Ideas]
Proposal for Examiner and Subject Profile System

As we continue to develop our device for training rats on specific tasks, I would like to propose a feature that enhances both usability and data management: the implementation of an Examiner and Subject Profile System.

Key Features:
Examiner Profile:

Credential Entry: Upon logging in, the examiner will input their credentials, allowing the system to tailor their dashboard to their specific needs and access level.
Personal Dashboard: The examiner will have a personalized dashboard where they can view and manage all their subjects (rats) and sessions. This dashboard will streamline their workflow by centralizing all relevant information.
Subject (Rat) Profile:

Health Profile Concept: Each subject will have a detailed health profile with a brief overview written by the examiner. This profile will allow for easy access to the subject's health history, behavioural data, and training progress.
Session Logs: All sessions related to the subject will be logged and easily accessible within this profile. This includes past, present, and scheduled sessions, along with any notes or observations made during these sessions.
Comprehensive Overview: The system will offer a complete overview of each subject's training journey, helping examiners quickly assess the rat's progress and make informed decisions.
Benefits:
Centralized Information: By having all relevant information in one place, examiners can quickly and efficiently manage their subjects and sessions.
Enhanced Data Accuracy: The ability to enter and review data directly within the profile minimizes errors and ensures all information is up-to-date.
Improved Workflow: This system will simplify the examiner's workflow, making it easier to track progress, document observations, and plan future sessions.

@ethierc  please take a look and give me your idea about this!

**Comment by ethierc** [comment]
I love it! Especially the part about keeping detailed experiment logs in a centralized place and organized by subjects.
For the experimenters profiles, I'm not sure I see the benefits. Since sometimes different people may be responsible to train individual animals, it might complicate things. I'm not sure.

**Reply by siamakhaz** [reply]
@ethierc Could you kindly reach out to those directly involved in the setup process to gather their feedback on this matter as well?

**Reply by ethierc** [reply]
Sure, we will discuss this at Monday's lab meeting. I'll let you know. Although if you were able to come, that could be great. Rudy will show us the current state of the system you two have put together. It will be at 1pm.
